modules/web/__api__/responses.py

```diff
@@ -39,6 +41,8 @@ def decorator(func):
 
 
 
-@response_for(lambda x: isinstance(x, dict))
-def dict_response(x):
-    return res.JSONResponse(x)
+@response_for(lambda x: isinstance(x, res.Response))
+def starlette_response(x):
+    return x
+
```

Comment:
Isn't this already handled by lines L24-25?

Reply:
Fixed